command/push.py

```diff
@@ -1,8 +1,32 @@
+import base64
+import os
+import requests
+import tarfile
+
 class PushCmd():
     name = 'push'
 
     def __init__(self):
         pass
         
-    def push(self):
-        print("Hello, world. The push feature is currently under construction.")
+    def push(self, bundle_dir, namespace, repository, release, auth_token):
```

Comment:
This method is public and should have docstring

Reply:
The intent is that this class (and so far all classes and methods implemented outside the cli.py file) should be considered "implementation details". I haven't really written python before, but based on what I am reading these classes should be marked with a single underscore? I can mark those that way, but it seems relatively obtuse from my perspective.

I'm currently working on a separate pull request that will implement an API that is intended to be called directly and will be well documented. That api will encapsulate the implementation details of all of these methods.

Reply:
In python there is really nothing private. Yeah Underscores means "private"